ports/windows/README.md

```diff
@@ -5,9 +5,12 @@ consider to contribute.
 
 
 Building on Debian/Ubuntu Linux system
+Building on Windows 10 with WSL (Windows Subsystem for Linux) Ubuntu Bash shell
 ---------------------------------------
 
     sudo apt-get install gcc-mingw-w64
+    make
```

Comment:
This is confusing. If this works, why would you ever use the line below?

Reply:
I added a default compiler because I want the same method to apply for all environments and it has to be just "make" for Windows which uses gcc.exe, size.exe, etc. But part of my philosophy has been to not change or break existing methods that people are happy with. I'm happy to remove the unnecessary alternative. If someone has scripts, makefiles etc that specify the CC they will still work so I guess it still backward compatible without documenting it.

Reply:
Sorry I missed the addition of a default for CROSS_COMPILE. I assume that is ok.